src/KIM/kim_interactions.cpp

```diff
@@ -66,9 +67,12 @@
 #include "input.h"
 #include "modify.h"
 #include "update.h"
+#include "fmt/format.h"
```

Comment:
see my commend for `kim_init`

Reply:
@akohlmey I updated the include statements